content/blog/2022-03-29-react-v18.md

```diff
@@ -86,7 +86,7 @@ setTimeout(() => {
   // React will render twice, once for each state update (no batching)
 }, 1000);
 
-// After: updates inside of timeouts, promises,
+// After: updates inside of timeouts, promises.
```

Comment:
It's "After: updates inside of timeouts, promises, native event handlers or any other event are batched." split across two lines.

Reply:
aha thanks for the explanation